### DISCUSSION TITLE
counter in Berry??

### DISCUSSION THREAD

**Discussion by trlafleur** [arendst/Tasmota#19208] [Q&A]
Are there any examples of using a counter in Berry??
I want to measure pulse width and duration from a sensor
Pulses are 10-50ms in width

thank you

**Comment by s-hadinger** [comment]
Did you look at the native counter feature? Pulses might be too short for polling and too long for blocking

**Comment by trlafleur** [comment]
I did, that why I ask the question, to see if it was possible...


Thanks...

~~ _/) ~~~~ _/) ~~~~ _/) ~~~~ _/) ~~

Tom Lafleur
